Source/WebKit/UIProcess/Cocoa/SOAuthorization/SOAuthorizationSession.mm

```diff
@@ -316,8 +318,9 @@ static bool isSameOrigin(const WebCore::ResourceRequest& request, const WebCore:
         return;
     }
 
-    m_page->websiteDataStore().cookieStore().setCookies(WTFMove(cookies), [this, weakThis = WeakPtr { *this }, response = WTFMove(response), data = adoptNS([[NSData alloc] initWithData:data])] () mutable {
-        if (!weakThis)
+    m_page->websiteDataStore().cookieStore().setCookies(WTFMove(cookies), [this, weakThis = ThreadSafeWeakPtr { *this }, response = WTFMove(response), data = adoptNS([[NSData alloc] initWithData:data])] () mutable {
```

Comment:
Same comment about capturing `this`

Reply:
This one needs this because of AUTHORIZATIONSESSION_RELEASE_LOG